### PR TITLE
Pin version for kubernetes provider across examples to exactly 1.10.0

### PR DESCRIPTION
**Pull request by andres-de-castro**
# PR o'clock

## Description

Semantic version fails on latest kubernetes provider,  pinning the version on examples exactly to 1.10.0 resolves many issues 

Please explain the changes you made here and link to any relevant issues.

### Checklist

- [ ] Change added to CHANGELOG.md. All changes must be added and breaking changes and highlighted
- [ ] CI tests are passing
- [ ] README.md has been updated after any changes to variables and outputs. See https://github.com/terraform-aws-modules/terraform-aws-eks/#doc-generation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# terraform-aws-eks
+r# terraform-aws-eks
```
Comment (confiq) [inline README.md]: typo?

**Comment by andres-de-castro** [comment]
> @andres-de-castro Thanks for opening this. Can you please update the changelog ?

For 8.0.1 ?

**Comment by barryib** [comment]
https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/CHANGELOG.md#v8---2020-xx-xx

**Comment by barryib** [comment]
> > @andres-de-castro Thanks for opening this. Can you please update the changelog ?
> 
> For 8.0.1 ?

https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/CHANGELOG.md#v8---2020-xx-xx

**Comment by andres-de-castro** [comment]
> > > @andres-de-castro Thanks for opening this. Can you please update the changelog ?
> > 
> > 
> > For 8.0.1 ?
> 
> https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/CHANGELOG.md#v8---2020-xx-xx

good to go!

**Pull request merged**